_config.yml

```diff
@@ -4,6 +4,7 @@ author: Code for Africa.
 
 url: https://bornperfect.africa
 enforce_ssl: bornperfect.africa
+image: /img/background/Header-banner.jpg
```

Comment:
@nerdlet Why would we put this here? The label/variable is also not very descriptive.

Reply:
@DavidLemayian I was creating a url image path for the site image.Already foxed it but forgot to remove that line.